crates/geometry/src/arc.rs

```diff
@@ -83,32 +81,27 @@ impl<Frame> Arc<Frame> {
     }
 
     pub fn length(&self) -> f32 {
-        let vector_start = self.start - self.circle.center;
-        let vector_end = self.end - self.circle.center;
+        let angle = self.start.angle_to(self.end, self.direction);
 
-        let angle_x_axis_to_start = vector_start.y().atan2(vector_start.x());
-        let mut angle = vector_end.y().atan2(vector_end.x()) - angle_x_axis_to_start;
+        angle.0 * self.circle.radius
+    }
 
-        if (self.direction == Direction::Clockwise) && (angle > 0.0) {
-            angle -= TAU;
-            angle *= -1.0;
-        }
-        if (self.direction == Direction::Counterclockwise) && (angle < 0.0) {
-            angle += TAU;
-        }
-        (angle * self.circle.radius).abs()
+    pub fn start_point(&self) -> Point2<Frame> {
+        self.circle.point_at_angle(self.start)
```

Comment:
Even if currently not used, I think the existence of this function should imply the existence of an `end_point` function.